### PR TITLE
Copied service endpoints specs to 2017-08-01

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by DeepakRajendranMsft — APPROVED** [review]

**Comment by DeepakRajendranMsft** [comment]
@sergey-shandar  could you also please take a look. this is the same PR as https://github.com/Azure/azure-rest-api-specs/pull/1533
we are porting the changes to 2017-08-01
Thanks!

File: specification/network/resource-manager/Microsoft.Network/2017-08-01/virtualNetwork.json [hunk 1]
```diff
@@ -825,6 +826,13 @@
           "$ref": "./routeTable.json#/definitions/RouteTable",
           "description": "The reference of the RouteTable resource."
         },
+        "serviceEndpoints": {
```
Comment (sergey-shandar) [inline specification/network/resource-manager/Microsoft.Network/2017-08-01/virtualNetwork.json]: This's breaking changes, same as here https://github.com/Azure/azure-rest-api-specs/pull/1533

**Comment by sergey-shandar** [comment]
@ravbhatnagar It's the same as https://github.com/Azure/azure-rest-api-specs/pull/1533 and contains the same breaking changes

**Comment by DeepakRajendranMsft** [comment]
@ravbhatnagar can you signoff on this again please, not sure why it is required but please do it.
@salameer this is just an another example of more processes that we have in the system. this change has already been approved by the ARM team for 06-01, not sure why we need their sign off again for 08-01

**Comment by ravbhatnagar** [comment]
@sergey-shandar  Wasnt this already signed off? ARM signs off.

**Comment by sergey-shandar** [comment]
@ravbhatnagar for different API version.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**